Examples/TicTacToe/Sources/Views-UIKit/LoginViewController.swift

```diff
@@ -137,7 +137,7 @@ class LoginViewController: UIViewController {
         guard let alert = alert else { return }
 
         let alertController = UIAlertController(
-          title: alert.title.formatted(), message: nil, preferredStyle: .alert)
+          title: alert.title.debugOutput, message: nil, preferredStyle: .alert)
```

Comment:
A couple reasons we may prefer #318:

1. Less reflection
2. UIKit interoperability (which we should maybe remove, example-wise). I've made `formatted()` internal in this PR since it seems weird to extend Apple's types with a method when it doesn't use any types we own.